spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/onnx/Gather.scala

```diff
@@ -17,12 +17,14 @@
 
 package com.intel.analytics.bigdl.nn.onnx
 
-import com.intel.analytics.bigdl.nn.ops
-import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
 import scala.reflect.ClassTag
+import com.intel.analytics.bigdl.nn
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
 
 
 object Gather {
-  def apply[T: ClassTag, D: ClassTag]()(implicit ev: TensorNumeric[T], ev2: TensorNumeric[D]):
-  ops.Gather[T, D] = new ops.Gather()
+  def apply[T: ClassTag, D: ClassTag](
+    axis: Int = 0
```

Comment:
add one parameter,but where it is used?

Reply:
Currently, BigDL ops.Gather does not support this parameter. In order to support it, we need to enrich the layer or implement another one, which I consider to do in next step.